src/unity/toolkits/object_detection/one_shot_object_detection/util/quadrilateral_geometry.cpp

```diff
@@ -6,16 +6,13 @@
 
 #include <unity/toolkits/object_detection/one_shot_object_detection/util/quadrilateral_geometry.hpp>
 
-static const boost::gil::rgb8_pixel_t BLACK(0, 0, 0);
-static const boost::gil::rgb8_pixel_t WHITE(255, 255, 255);
+static const size_t TRANSPARENT(0);
+static const size_t OPAQUE(255);
```

Comment:
Nit: for simple types like size_t, you should be able to use `constexpr` (making these essentially compile-time constants with no allocated memory at runtime), and to use the clearer `=` syntax, e.g.
```
static constexpr size_t TRANSPARENT = 0;
```

Reply:
Fixed!